static/js/util/profile_edit.js

```diff
@@ -312,22 +312,22 @@ export function boundMonthYearField(keySet, label) {
   // fullWidth means set width = 100% instead of 256px
   return <div>
     <label>{label}</label> <TextField
-    floatingLabelText="MM"
-    style={{
-      maxWidth: "2em"
-    }}
-    fullWidth={true}
-    value={monthYear.month !== undefined ? monthYear.month : ""}
-    onChange={e => setNewDate(e.target.value, undefined)}
-  /> <span className="slash">/</span> <TextField
-    floatingLabelText="YYYY"
-    style={{
-      maxWidth: "4em"
-    }}
-    fullWidth={true}
-    value={monthYear.year !== undefined ? monthYear.year : ""}
-    onChange={e => setNewDate(undefined, e.target.value)}
-  />
+      floatingLabelText="MM"
+      style={{
+        maxWidth: "2em"
+      }}
+      fullWidth={true}
+      value={monthYear.month !== undefined ? monthYear.month : ""}
+      onChange={e => setNewDate(e.target.value, undefined)}
+    /> <span className="slash">/</span> <TextField
+      floatingLabelText="YYYY"
+      style={{
+        maxWidth: "4em"
+      }}
+      fullWidth={true}
+      value={monthYear.year !== undefined ? monthYear.year : ""}
+      onChange={e => setNewDate(undefined, e.target.value)}
+    />
```

Comment:
I think this may just be a differing style preference, but I find JSX a lot easier to read when the opening and closing tags are generally one per line (obviously excepting very short tags like `<span>{ alertText() }</span>`).